locales/ita.json

```diff
@@ -0,0 +1,99 @@
+  "message-hubs": "Sensor.Community ha hub globali a Stoccarda, Sofia e Varsavia."
+  },
+  "sensor": {
+    "metaDescription": "",
+    "metaTitle": "Trova il tuo sensore e le istruzioni di costruzione faidate per il tuo sensore di polvere fine",
+    "h1": "Sensori in kit",
+    "subtitle": " Trova le nostre guide per il tuo sensore"
+  },
+  "guide": {
+    "editOnGithub": "Modifica su Github"
+  },
+  "airrohr": {
+    "metaTitle": "Costruisci il tuo sensore di particolato.",
+    "metaDescription": ""
+  },
+  "forum": {
+    "metaDescription": "",
+    "metaTitle": "Forum",
+    "h1": "Forum",
+    "subtitle": "Prossimamente",
+    "getNotified": "\uD83D\uDC8C Sottoscrivi"
+  },
+  "donate": {
+    "metaDescription": "Siamo senza fini di lucro, non aziendali e senza compromessi. Le persone come te ci aiutano. Facciamo affidamento su donazioni per svolgere la nostra missione.",
+    "metaTitle": "Dona - aiutaci mostrando il tuo affetto",
+    "h1": "Mostra il tuo supporto",
+    "description": "Siamo senza fini di lucro, non aziendali e senza compromessi. Le persone come te ci aiutano. Facciamo affidamento su donazioni per svolgere la nostra missione.",
+    "section": "Modi di supportare",
+    "campaign": "Campagna in corso",
+    "donation": "Donazione diretta",
+    "holder": "Titolare del conto",
+    "iban": "IBAN",
+    "bic": "BIC",
+    "bank": "Banca",
+    "intendedUse": "Causale",
+    "donationDescription": "The Open Knowledge Foundation Germany é riconosciuta come una organizzazione non-profit."
+  },
+  "blog": {
+    "metaDescription": "News",
+    "metaTitle": "News",
+    "h1": "Posts recenti",
+    "description": " Trova news o altre informazioni interessanti"
+  },
+  "contributors": {
+    "metaDescription": "Le persone dietro al progetto",
+    "metaTitle": "Collaboratori di Sensor.Community",
+    "h1": "Ciao dal team \uD83D\uDC4B",
+    "description": "Incontra il team che sta lavorando dietro le quinte per sfruttare la potenza di Sensor.Community.",
+    "contactBottom": "\uD83D\uDCEE Per favore usa l'opzione contatti in fondo alla pagina",
+    "meetTeam": "Incontra il team"
+  },
+  "projects": {
+    "metaDescription": "Trova app mobili, siti Web, ... dalla nostra adorabile community \uD83D\uDC96",
+    "metaTitle": "Progetti della Community ",
+    "h1": "Progetti della Community",
+    "description": "Trova app mobili, siti Web, ... dalla nostra adorabile community \uD83D\uDC96?",
+    "submitIdea": "Hai un grande progetto? Condividilo con la community!",
+    "submitProject": "Invia il tuo progetto ?"
+  },
+  "hubs": {
+    "metaDescription": "Trova comunitá in tutto il mondo \uD83D\uDC96?",
+    "metaTitle": "Community Hubs",
+    "h1": "Community Hubs",
+    "description": "Trova comunitá in tutto il mondo \uD83D\uDC9??
```

Comment:
Hi @flavio045, thanks for your translation. I found some errors:
* can you rename the file to `ìt.json` instead of  `ita.json`?
* there are some missing quotes at the end of the file
* missing chars for the emojis, there were replaced with question marks for some reason
* missing parts of the translation
* therefore missing closing swirly brackets `}`to close the json

Reply:
I'm not so use to GitHub: I made some error uploading/copying text